yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/block/TestBlock.java

```diff
@@ -101,6 +104,7 @@ public final class TestBlock extends ConnectedBlock {
     static final String OPTION_CONNECTION_LIFECYCLE = "connection_lifecycle";
     static final String OPTION_CONNECTION_LIFECYCLE_TEST = "test";
     static final String OPTION_CONNECTION_LIFECYCLE_BLOCK = "block";
+    static final String OPTION_CONNECTION_OPTIONS = "connection_options";
```

Comment:
Should this go into `FileOption`s as well? At this point it may be late, but it would be nice to be able to set these kinds of things at a file level